tests/lib/Layout/Resolver/Form/ConditionType/Mapper/ResourceTypeTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Tests\Layout\Resolver\Form\ConditionType\Mapper;
+
+use Netgen\Layouts\Sylius\Layout\Resolver\Form\ConditionType\Mapper\ResourceType;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+
+#[CoversClass(ResourceType::class)]
+final class ResourceTypeTest extends TestCase
+{
+    private ResourceType $mapper;
+
+    protected function setUp(): void
+    {
+        $allowedResources = [
+            'Sylius\Component\Product\Model\ProductInterface' => 'product',
```

Comment:
Use `::class` constants here, please.

Reply:
Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/11/commits/93ab79cc5e0f027f9222331aa0551e4358fa80db